cli/types/cypress.d.ts

```diff
@@ -792,7 +819,35 @@ declare namespace Cypress {
     clearCookies(options?: CookieOptions): Chainable<null>
 
     /**
-     * Clear data in local storage.
+     * Get local storage for all origins.
+     *
+     * @see https://on.cypress.io/getalllocalstorage
+     */
+    getAllLocalStorage(options?: Partial<Loggable>): Chainable<StorageByOrigin>
+
+    /**
+     * Clear local storage for all origins.
+     *
+     * @see https://on.cypress.io/clearalllocalstorage
+     */
+    clearAllLocalStorage(options?: Partial<Loggable>): Chainable<null>
```

Comment:
If this shouldn't be chained further, isn't the correct return type just null?
https://github.com/cypress-io/cypress-documentation/pull/4876/files#diff-cfb67e9dbdc66c73104dbe980789350ec2bc60eadaf0471868ab1c7937c43902R44

Reply:
Even commands that _shouldn't_ be chained further still return chainers and technically _can_ be chained off of. They just yield `null` instead of a subject.

Reply:
We should prob drop the statement that it cant' be chained further then. It's misleading if the types don't reflect the docs.

Reply:
Other commands that yield null say that, so I added it for consistency, but I agree it's misleading and doesn't really add any benefit, so I'll remove it from the docs.